docs/reports/reports.rst

```diff
@@ -88,9 +88,9 @@ The parent data source provides a high-level summary of the data while the child
   
 * **Published** - Select **Yes** to ensure that the Report is active, ready to send when scheduled in the Report options.
 
-* **System Report** - Select **Yes** so that all Users can access the Report. If set to No, only the owner of the Report and Users with permission to view others' Reports can see the Report.
+* **Visible for all logged-in users** - Select **Yes** so that all Users can access the Report. If set to No, only the owner of the Report and Users with permission to view others' Reports can see the Report.
```

Comment:
```suggestion
* **Visible for all logged-in Users** - Select **Yes** so that all Users can access the Report. If set to No, only the owner of the Report and Users with permission to view others' Reports can see the Report.
```

Reply:
Is this a standard?
Or we are trying to make it a standard?

Reply:
In the docs we always capitalise features of the product - we apply the Google Technical Writing Style Guide: https://developers.google.com/style - although it seems there was a recent update which now says _not_ to capitalise features where it used to say that you should!

For the time being we stick with what we currently use.